### PR TITLE
Flask into docker

### PR DESCRIPTION
**Pull request by Zoxon470**
#12

### CODE REVIEW

**Comment by dmi3coder** [comment]
Good day @Zoxon470! I love the structure of files that you've done. One thing though is that `./client` folder is a [ react submodule](https://github.com/donautech/behaiv-remote-kernel-client). React will compile into static files by `client_dev.sh` and being copied over to `/app/static` folder.
So I think it'll be great to move docker files to root or somewhere near, not in a client folder
Is it better to re-name such a folder? Maybe I'm not aware that the `client` name is responsible for something else in flask/python. What about `webapp`?
Thank you for a PR :)

File: client/flask/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM python:3.7.4-alpine
```
Comment (dmi3coder) [inline client/flask/Dockerfile]: I think the `client` folder is dedicated to react app

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/donautech/behaiv-remote-kernel/issues"
+  },
+  "homepage": "https://github.com/donautech/behaiv-remote-kernel#readme",
+  "dependencies": {
+    "bootstrap": "^4.3.1"
+  }
+}
```
Comment (dmi3coder) [inline package.json]: What is it used for? Just to track versioning?

**Review by dmi3coder — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Zoxon470) [reply package.json]: This is just an example for the npm install command to work.

File: client/flask/Dockerfile (hunk 1 above)
Comment (Zoxon470) [reply client/flask/Dockerfile]: Ok. I have changed it. I've created directory `docker`. This directory is for docker setup and your flask and react apps

**Comment by dmi3coder** [comment]
Looks great👍, I'll test now

**Comment by dmi3coder** [comment]
@Zoxon470 looks really great, I made some simplifications for docker folder and added crutial `--host 0.0.0.0` for start.sh. Without it, it will serve on containers 127.0.0.1 which is not available outside of containers. Also added `./client_dev.sh`. now when you go to 0.0.0.0:5000 it'll open our UI
Love the stat function that re-starts the project on change, this will boost the development process a lot.
Thank you for help!

**Pull request merged**